src/app/api/projects/[slug]/route.ts

```diff
@@ -0,0 +1,81 @@
+import db from "@/db"
+import { featureCollection, lineString } from "@turf/helpers"
+
+export async function GET(request: Request, { params }: { params: Promise<{ slug: string }> }) {
+  try {
+    const slug = (await params).slug
+    const project = await db.project.findFirst({
+      where: { slug },
+      select: {
+        isExportApi: true,
+      },
+    })
+
+    if (!project) {
+      console.error(`No project found for slug: ${slug}`)
```

Comment:
```suggestion
      console.log('No project found for slug:', slug)
```

Ich glaube, in den Logs sehen die ".log" besser aus als die ".error", weil .error den Stacktrace enthält der für diese Info gar nicht wichtig ist.

Randnotiz: Bei console.* kann man ganz übersichtlich eine komma-liste übergeben.